docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,97 @@
+	- set the value to expected_system_type_list: `zones_and_systems[zone] = expected_system_type_list`
+	
+- now we need to go through each 'exception' to Table_G3_1_1 in order.
+
+- G3.1.1b "Use additional system types for nonpredominant conditions (i.e., residential/nonresidential) if those conditions apply to more than 20,000 ft2 of conditioned floor area." uses the same logic that we just used for the expected_system_type_list, except that we select systems based on non-predominant building area types and the area of these non-predominant conditions
+- loop throught the building area types: `for bat in list_building_area_types_and_zones:`
+	- look for any bat's that have more than 20000 ft2 area: `if list_building_area_types_and_zones[bat]["AREA"] >=20000:`
+		- we select a secondary system type for this bat: `secondary_system_type = expected_system_type_from_Table_G3_1_1(bat,num_floors,list_building_area_types_and_zones[bat]["AREA"],climate_zone)`
+		- change the system type for any zone in this bat: `for zone in zones_and_systems:`
+			- check if this zone is in the list of zone_ids associated with this bat: `if zone.id in list_building_area_types_and_zones[bat]["ZONE_IDS"]:`
+				- change the system type to: [secondary_system_type[0], "G3_1_1b"] - remember that expected_system_type_from_Table_G3_1_1 gives both the expected system type and a string explaining how that system type was chosen.  For the rule testing G3_1_1b, it is only asking whether the system was chosen from Table G3.1.1, not exactly how it got there: `zones_and_systems[zone] = [secondary_system_type[0], "G3_1_1b"]`
+
+- G3.1.1c "If the baseline HVAC system type is 5, 6, 7, 8, 9, 10, 11, 12, or 13 use separate single-zone systems conforming with the requirements of system 3 or system 4 for any HVAC zones that have occupancy, internal gains, or schedules that differ significantly from the rest of the HVAC zones served by the system. The total peak internal gains that differ by 10 Btu/h·ft2 or more from the average of other HVAC zones served by the system, or schedules that differ by more than 40 equivalent full-load hours per week from other spaces served by the system, are considered to differ significantly. Examples where this exception may be applicable include but are not limited to natatoriums and continually occupied security areas. This exception does not apply to computer rooms."
+- loop through the zones_and_systems to see if any of the zones meets Ge.1.1.c: `for zone in zones_and_systems:`
+	- use the function does_zone_meet_G3_1_1c to determine if this zone meets this requirement: `if does_zone_meet_G3_1_1c(B_RMI,zone,zones_and_systems[zone][0] == YES):`
+		- the zone meets the G3_1_1c requirements, choose system 3 or 4 based on climate zone: `if is_CZ_0_to_3a(climate_zone):`
+			- set the system to "SYS-3" and source to "G3_1_1c": `zones_and_systems[zone] = ["SYS-3",zones_and_systems[zone][0] + " G3_1_1c"]`
```

Comment:
Is this a string or a list assigning to the key?

Reply:
there are two strings, the first is the expected system type "SYS-3", the second is a string that will be matched to the `target_string` for the individual rules.

However, I've reformatted this to be a dict instead of a paired list